src/java/org/apache/cassandra/service/accord/txn/TxnCondition.java

```diff
@@ -532,7 +589,13 @@ else if (type.isUDT())
         @Override
         public boolean applies(TxnData data)
         {
-            return getBounds(data).appliesTo(reference.getRow(data));
+            Bound bounds = getBounds(data);
```

Comment:
nit: OCD in me is wondering if we can just change the method to singular `getBound()`